src/server/generic_family.h

```diff
@@ -38,42 +32,40 @@ class GenericFamily {
   static OpResult<uint32_t> OpDel(const OpArgs& op_args, const ShardArgs& keys);
 
  private:
-  static void Del(CmdArgList args, ConnectionContext* cntx);
-  static void Ping(CmdArgList args, ConnectionContext* cntx);
-  static void Exists(CmdArgList args, ConnectionContext* cntx);
-  static void Expire(CmdArgList args, ConnectionContext* cntx);
-  static void ExpireAt(CmdArgList args, ConnectionContext* cntx);
-  static void Persist(CmdArgList args, ConnectionContext* cntx);
-  static void Keys(CmdArgList args, ConnectionContext* cntx);
-  static void PexpireAt(CmdArgList args, ConnectionContext* cntx);
-  static void Pexpire(CmdArgList args, ConnectionContext* cntx);
-  static void Stick(CmdArgList args, ConnectionContext* cntx);
-  static void Sort(CmdArgList args, ConnectionContext* cntx);
-  static void Move(CmdArgList args, ConnectionContext* cntx);
-
-  static void Rename(CmdArgList args, ConnectionContext* cntx);
-  static void RenameNx(CmdArgList args, ConnectionContext* cntx);
-  static void ExpireTime(CmdArgList args, ConnectionContext* cntx);
-  static void PExpireTime(CmdArgList args, ConnectionContext* cntx);
-  static void Ttl(CmdArgList args, ConnectionContext* cntx);
-  static void Pttl(CmdArgList args, ConnectionContext* cntx);
+  using SinkReplyBuilder = facade::SinkReplyBuilder;
 
-  static void Echo(CmdArgList args, ConnectionContext* cntx);
-  static void Select(CmdArgList args, ConnectionContext* cntx);
-  static void Scan(CmdArgList args, ConnectionContext* cntx);
-  static void Time(CmdArgList args, ConnectionContext* cntx);
-  static void Type(CmdArgList args, ConnectionContext* cntx);
-  static void Dump(CmdArgList args, ConnectionContext* cntx);
-  static void Restore(CmdArgList args, ConnectionContext* cntx);
-  static void RandomKey(CmdArgList args, ConnectionContext* cntx);
-  static void FieldTtl(CmdArgList args, ConnectionContext* cntx);
-  static void FieldExpire(CmdArgList args, ConnectionContext* cntx);
+  static void Del(CmdArgList args, Transaction* tx, SinkReplyBuilder* builder);
```

Comment:
now we are going to pass `Transaction* tx, SinkReplyBuilder* builder` explicitly.